libs/formAnswers.js

```diff
@@ -103,24 +103,31 @@ function getMultipleFieldAnswers(field, previousAnswers) {
   return answers;
 }
 
-function getInitialValue(field, user, previousAnswers) {
+function getInitialValue(field, users, previousAnswers) {
   let initialValue;
 
   field.loadPrevious.forEach(matchString => {
+    let user;
     const strArray = matchString.split('.');
-    if (strArray[0] === 'user') {
-      initialValue = getUserInfo(user, strArray.slice(1)) || initialValue;
-      if (initialValue) {
-        return formatAnswer(field.id, field.tags, initialValue);
-      }
+    switch (strArray[0]) {
```

Comment:
Is there a reason for not destructing this value further up? ie ´const [value] = matchString.split(' ')´